frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/premium.embedding.unit.spec.tsx

```diff
@@ -41,22 +49,50 @@ describe("[EE, with token] embedding settings", () => {
       });
 
       it("should not prompt to upgrade to remove the Powered by text", async () => {
-        await setupPremium({
-          settingValues: { "enable-embedding": false },
-        });
-
         expect(
           screen.queryByText("upgrade to a paid plan"),
         ).not.toBeInTheDocument();
       });
     });
 
+    describe("embedding SDK", () => {
+      it("should show info about embedding SDK", async () => {
+        const withinEmbeddingSdkCard = within(
+          screen.getByRole("article", {
+            name: "Embedding SDK for React",
+          }),
+        );
+
+        expect(
+          withinEmbeddingSdkCard.getByRole("heading", {
+            name: "Embedding SDK for React",
+          }),
+        ).toBeInTheDocument();
+        expect(
+          withinEmbeddingSdkCard.getByText(
+            /Interactive embedding with full, granular control./,
+          ),
+        ).toBeInTheDocument();
+      });
+
+      it("should allow access to the embedding SDK settings page", async () => {
+        // Go to embedding SDK settings page
+        await userEvent.click(
+          within(
+            screen.getByRole("article", {
+              name: "Embedding SDK for React",
+            }),
+          ).getByRole("button", { name: "Configure" }),
+        );
+
+        expect(
+          screen.getByLabelText("Cross-Origin Resource Sharing (CORS)"),
+        ).toBeEnabled();
```

Comment:
This test will be changed to test that it's disabled when `enable-embedding-sdk` setting is introduce in the next PR.